.github/workflows/check-deprecated-file-change.yaml

```
@@ -0,0 +1,20 @@
+name: check-deprecated-file-change
+
+on:
+  pull_request:
+
+jobs:
+  check-deprecated-file-change:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Check deprecated file change
+        run: |
+          if ! git diff --exit-code --quiet "origin/${{ github.base_ref }}...HEAD" -- cspell/.cspell.json; then
+            echo "Don't edit 'cspell/.cspell.json', edit '.cspell.json' instead."
```

How about adding a warning message?
```suggestion
            echo "::warning file=cspell/.cspell.json::Don't edit 'cspell/.cspell.json', edit '.cspell.json' instead."
```

---

Thank you! It's nice. :+1: